bots/notify/src/main/java/org/openjdk/skara/bots/notify/prbranch/PullRequestBranchNotifier.java

```diff
@@ -69,13 +65,17 @@ private void deleteBranch(PullRequest pr) {
         if (protectBranches) {
             log.info("Removing branch protection for " + branch);
             pr.repository().unprotectBranchPattern(branch);
```

Comment:
We still need this because it's possible that we have some pr branch protected before we deploy this patch.

Reply:
You should add a code comment explaining this so we may eventually remove that code later.

Have you checked how the call behaves if there is no protection to remove?

Reply:
Sure, will add the comment. No, I didn't check it by myself because I saw your test in `GitLabRestApiTest::branchProtection`. I will check it again later.

Reply:
Checked it again, if there is no protection to remove, nothing would happen. The bot works fine.